xml/System.IO.Pipelines/PipeReader.xml

```diff
@@ -97,8 +98,11 @@ The memory for the consumed data will be released and no longer available.
 
 The <xref:System.IO.Pipelines.ReadResult.Buffer> previously returned from <xref:System.IO.Pipelines.PipeReader.ReadAsync(System.Threading.CancellationToken)> must not be accessed after this call.
 
+This is equivalent to calling <xref:System.IO.Pipelines.AdvanceTo(System.SequencePosition, System.SequencePosition)> with identical examined and consumed positions.
 The examined data communicates to the pipeline when it should signal more data is available.
 
+Because the consumed parameter doubles as the examined parameter, the consumed parameter should be greater than or equal to the examined position in the previous call to AdavanceTo. Otherwise <xref:System.InalidOperationException> will be thrown.
```

Comment:
```suggestion
Because the consumed parameter doubles as the examined parameter, the consumed parameter should be greater than or equal to the examined position in the previous call to `AdvanceTo`. Otherwise, an <xref:System.InvalidOperationException> is thrown.
```